tests/functions/void_parameter.phpt

```diff
@@ -0,0 +1,8 @@
+--TEST--
+void return type: not valid as a parameter type
+--FILE--
+<?php
+
+function foobar(void $a) {}
+--EXPECTF--
+Fatal error: void cannot be used as a parameter type in %s on line %d
```

Comment:
Not sure if this test should be included, since a reasonable implementation might consider this a syntax error.

(Also, should we really be testing error messages anyway? They're not in the spec. :/)